SRC/la_constants.f90

```diff
@@ -119,10 +120,11 @@ module LA_CONSTANTS
        (minexponent(0._dp) - 1) * 0.5_dp)
    real(dp), parameter :: dtbig = real(radix(0._dp), dp)**floor( &
        (maxexponent(0._dp) - digits(0._dp) + 1) * 0.5_dp)
-!  ssml = 1/s, where s was defined in https://doi.org/10.1145/355769.355771
+!  ssml >= 1/s, where s was defined in https://doi.org/10.1145/355769.355771
+!  The correction was added in https://doi.org/10.1145/3061665 to scale denormalized numbers correctly 
    real(dp), parameter :: dssml = real(radix(0._dp), dp)**( - floor( &
-       (minexponent(0._dp) - 1) * 0.5_dp))
-!  ssml = 1/S, where S was defined in https://doi.org/10.1145/355769.355771
+       (minexponent(0._dp) - digits(0._dp)) * 0.5_dp))
+!  sbig = 1/S, where S was defined in https://doi.org/10.1145/355769.355771
    real(dp), parameter :: dsbig = real(radix(0._dp), dp)**( - ceiling( &
```

Comment:
I'm still unsure that this formula is correct. It gives `dsbig=5.0052077379577523E-147` while the Anderson's code uses the Blue's formula which gives `1.1113793747425387E-162`. Further, some test cases in my Go port overflow with this `dsbig` value but do not with Anderson's value. It seems that the part `- digits(0._dp) + 1` should be `+ digits(0._dp) - 1` to reproduce the value from Anderson's code.

Reply:
Right! I will fix it. Thanks again!

Reply:
Hi @vladimir-ch. Could you please review my changes once more. I think now everything is fine. Thanks!

Reply:
Yes, LGTM. Thanks!